masonite/middlewares/VerifyCsrfToken.py

```diff
@@ -0,0 +1,33 @@
+from masonite.exceptions import InvalidCSRFToken
+
+
+class VerifyCsrfToken(object):
+    """
+    Verify csrfk token middleware
+    """
+
+    exempt = []
+
+    def __init__(self, Request):
+        self.request = Request
+
+    def befofe(self):
+        request = self.request
+        if request.is_post():
+            token = request.input('csrftoken')
+            if not request.verify_csrf_token(token) and not self.__in_except():
```

Comment:
When you move these verify methods into it's own `CSRF` class make sure you use that here.